system-apps/web-browser-app/webClient/src/app/browser/address-bar/address-bar.component.scss

```diff
@@ -0,0 +1,52 @@
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+
+  SPDX-License-Identifier: EPL-2.0
+
+  Copyright Contributors to the Zowe Project.
+*/
+
+:host {
+  flex: 0 0 auto;
+  display: flex;
+  align-items: center;
+  overflow: hidden;
+  font-size: 1.2rem;
+  border-bottom: 1px solid #777777;
+}
+
+input[type="url"] {
+  flex: 1 1 auto;
+}
+
+button {
+  flex: 0 0 3rem;
+}
+
+.icon-button {
+  font-weight: 600;
+}
+
+.proxy {
+  flex: 0 0 auto;
+  width: 6rem;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 0 0.5rem;
+  
+  label {

```

Comment:
The same as above. This must be safe.

Reply:
It is safe because Angular adds a unique attribute at runtime to the element and limits the scope  for the CSS rules like this:
![image](https://user-images.githubusercontent.com/34669449/79458215-f106bb80-800a-11ea-8598-56a6c7162323.png)